perception/autoware_tensorrt_yolox/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
 cmake_minimum_required(VERSION 3.17)
 project(autoware_tensorrt_yolox)
 
-find_package(tensorrt_common)
-if(NOT ${tensorrt_common_FOUND})
+find_package(autoware_tensorrt_common)
+if(NOT ${autoware_tensorrt_common_FOUND})
```

Comment:
nit: the warning still says `tensorrt_common` package

Reply:
(could not leave a comment on the actual line, since it was not a change on this PR)

Reply:
Ehh, I missed it. Thank you Kenzo!
Adressed in https://github.com/autowarefoundation/autoware.universe/pull/9099/commits/b3596304d19a8fd05377babe8fdf572bb24fa3b5.